upstairs/src/dummy_downstairs_tests.rs

```diff
@@ -128,7 +126,7 @@ impl DownstairsHandle {
         self.tx.send(m)
     }
 
-    pub async fn negotiate_start(&mut self) -> Result<()> {
+    pub async fn negotiate_start(&mut self) {
```

Comment:
Since this is a sub-function, and not a test itself, I thought would it better for this to 
return error and have the calling funciton panic.  If it panics here, then I get a 
`line 149: read only mismatch`, but then I have to figure out where in my test I called `
negotiate_start()`.  However, after going through the places that call
`negotiate_start()`, it does seem pretty obious where they come from so I think
it's okay that we just panic in this method.

Reply:
Also, we get a full backtrace with `RUST_BACKTRACE=1` 😉